plugins/scaffolder-backend/src/lib/logger.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as winston from 'winston';
```

Comment:
Making a common one is hard unless we wanna have `backend-common` or something because of `.cjs` ahah I just tried and got a load of errors in the client because I could open a write to `stdout`

Reply:
I've been toying with the idea of explicit injection of e.g. logger and other basics. E.g. that they essentially are passed as arguments by the backend service when instantiating the plugin

Reply:
`type DefaultBackendConfig = { logger: Logger }; type MyConfig = { serviceUrl: string; } & DefaultBackendConfig;`

Reply:
#686